experimental/index.html

```diff
@@ -41,17 +47,74 @@
         };
         datacommons.drawRanking(element1, props1);
         datacommons.drawRanking(element2, props2);
+
+        var lineChartElement1 = document.getElementById("my-line-chart-1");
+        var lineChartElement2 = document.getElementById("my-line-chart-2");
+        var lineChartProps1 = {
+          id: "my-line-chart-content-1",
```

Comment:
I thought props.id should be the same as the html element id, but looks like it works here.

Reply:
It looks like this `id` gets applied to the component's inner div that contains the chart svg 

![Screenshot 2023-05-25 at 1 08 17 PM](https://github.com/datacommonsorg/website/assets/13766/400159b4-1a6a-43ff-83b6-9198239ade58)